vignettes/tcplfit2-vignette.Rmd

```diff
@@ -302,11 +307,25 @@ myfun <- function(y) {
                           )
   list(list(res)) #use list twice because data.table uses list(.) to look for values to assign to columns
 }
+```
 
+**Warning: The fitting step for the full data set, `dat`, can take 7-10 minutes to run.** Hence we will provide a subset of data for curve fitting and hitcalling. The subset data only contains records of six samples.
```

Comment:
Update this block of text to
"The following code performs dose-response modeling for all spids in the dataset.  Warning <...> Hence the code chunk following provides a subset example of data for curve fitting and hitcalling.  <...>"
The <...> keeps the rest of the text as is.

Reply:
the text has been updated